hydrolearn-theme/lms/templates/discovery/course_card.underscore

```diff
@@ -38,7 +38,7 @@
             
             <div class='course-weekly-effort' aria-hidden="true">
                 <% if (typeof(effort) != "undefined") { %>
-                    <%- gettext("Effort:") %> <%- effort %> <%- gettext("hrs/wk") %>
+                    <%- gettext("Effort:") %> <%- effort %> <%- gettext("hours") %>
```

Comment:
This is the piece I was not able to get a test for.  It'll show on the course card of the courses page